bindings/python/test/test_bson.py

```diff
@@ -154,3 +154,24 @@ def test_object_id_type(self):
         arr = pa.ExtensionArray.from_storage(ObjectIdType(), storage_array)
         result = self.serialize_array(arr)
         assert result.type._type_marker == ObjectIdType._type_marker
+
+
+class TestBooleanType(TestBsonToArrowConversionBase):
+    def setUp(self):
+        self.schema = Schema({'_id': ObjectId,
+                              'data': bool_()})
+        self.context = PyMongoArrowContext.from_schema(
+            self.schema)
+
+    def test_simple(self):
+        ids = [ObjectId() for i in range(4)]
+        docs = [{'_id': ids[0], 'data': True},
+                {'_id': ids[1], 'data': False},
+                {'_id': ids[2], 'data': True},
+                {'_id': ids[3], 'data': False}]
```

Comment:
Please test missing and non-bool 'data' fields as well to test the append_null behavior of `process_bson_stream`.

Reply:
Done.